runtimes/politest/src/lib.rs

```diff
@@ -835,9 +838,11 @@ where
 			frame_system::CheckTxVersion::<Runtime>::new(),
 			frame_system::CheckGenesis::<Runtime>::new(),
 			frame_system::CheckEra::<Runtime>::from(generic::Era::mortal(period, current_block)),
-			frame_system::CheckNonce::<Runtime>::from(nonce),
+			pallet_dispenser::extensions::CheckNonce::<Runtime>::from(nonce),
```

Comment:
Is this a temporary measure needed until https://github.com/paritytech/polkadot-sdk/issues/3991 gets fixed? In that case maybe let's add a `TODO` so we remember to go back to `frame_system::CheckNonce::<Runtime>::from(nonce)` as soon it's possible